desktopVersion.css

```diff
@@ -0,0 +1,502 @@
+    display: flex;
+    gap: 22px;
+  }
+
+  .canopy__ul li {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+    list-style: disc;
+  }
+
+  .descr__paragr {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    padding-top: 0.7rem;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .skills {
+    padding-top: 1.2rem;
+    display: flex;
+    gap: 8px;
+  }
+
+  .skills__button {
+    display: flex;
+    background-color: #ebebff;
+    padding: 5px 12px;
+    height: 35px;
+    border-radius: 8px;
+    width: auto;
+  }
+
+  .btn__skills {
+    font-family: "Poppins", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    text-align: left;
+    color: #6070ff;
+    background: transparent;
+    border: none;
+  }
+
+  .skills__button,
+  .skills__button1 {
+    list-style: none;
+  }
+
+  .see__proj__btn {
+    padding: 12px;
+    border-radius: 8px;
+    width: 126px;
+    margin-top: 1.4rem;
+    background-color: transparent;
+    border: 1px solid #6070ff;
+  }
+
+  .see__proj__btn a {
+    font-family: "Poppins", sans-serif;
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    text-align: center;
+    color: #396df2;
+    cursor: pointer;
+  }
+
+  /* =======================END OF POST========================= */
+
+  #about {
+    position: relative;
+    background-color: white;
+    border-top-right-radius: 6rem;
+  }
+
+  .about__container {
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 5rem;
+    gap: 4%;
+    padding: 4rem;
+  }
+
+  .about__container_left {
+    width: 42%;
+  }
+
+  .about__container_right {
+    width: 54%;
+  }
+
+  .about__container h1 {
+    font-family: "Poppins", sans-serif;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: left;
+    color: #172b4d;
+  }
+
+  .about__container p {
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #344563;
+    margin-bottom: 1rem;
+  }
+
+  .let__g {
+    font-family: "Poppins", sans-serif;
+    font-size: 13px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #7f8cff;
+    margin-bottom: 0.4rem;
+  }
+
+  .social_media {
+    display: flex;
+    gap: 6px;
+    margin-top: 0.6rem;
+  }
+
+  .social__media__lin {
+    list-style: none;
+  }
+
+  .social__media__imag {
+    width: 23px;
+    height: 14px;
+    border-radius: 8px;
+  }
+
+  .resume {
+    padding: 0 2px;
+    display: flex;
+    justify-content: center;
+    border-radius: 8px;
+    width: 135px;
+    height: 48px;
+    margin-top: 1.4rem;
+    background-color: transparent;
+    border: 1px solid #6070ff;
+  }
```

Comment:
- [OPTIONAL] Kindly make the text inside your button in a single line as it is on the figma design.

### Your Design
![your-about](https://github.com/sibomana123/MV__Portfolio/assets/26597432/89bb373d-d012-409c-8304-b7b394d2d4f8)

### Figma Design 
![temp-about](https://github.com/sibomana123/MV__Portfolio/assets/26597432/cfbe872a-7704-4350-8bc4-7411aef4ccc5)